airflow/providers/google/cloud/operators/cloud_storage_transfer_service.py

```diff
@@ -1053,8 +1075,8 @@ def _create_body(self) -> dict:
             DESCRIPTION: self.description,
             STATUS: GcpTransferJobsStatus.ENABLED,
             TRANSFER_SPEC: {
-                GCS_DATA_SOURCE: {BUCKET_NAME: self.source_bucket},
-                GCS_DATA_SINK: {BUCKET_NAME: self.destination_bucket},
+                GCS_DATA_SOURCE: {BUCKET_NAME: self.source_bucket, PATH: self.source_path},
```

Comment:
Do you think its ok to pass `None` in here? From the docs: 

_"Must be an empty string or full path name that ends with a '/'. This field is treated as an object prefix. As such, it should generally not begin with a '/'."_

https://cloud.google.com/storage-transfer/docs/reference/rest/v1/TransferSpec#awss3data

If this is the case then maybe the default value for paths should be `""`, or we should be only adding `PATH` to this dictionary if it is not None.

Reply:
I've checked for both source and sink - Google's API correctly handles it omitting `None` args for path/prefix.

Reply:
Great, thanks for clarifying.